src/utils/common.ts

```diff
@@ -25,3 +25,9 @@ export function objectToQueryString(o: any): string {
 
   return qsResult ? `?${qsResult}` : ''
 }
+
+export function fromHexString(hexString: string) {
```

Comment:
Did the fn from multihash lib not work?
I added it in my example just to check whether or not it was the issue, but if that works no need to create this custom util

Reply:
It's not from `multihash` lib. I used [uint8arrays](https://www.npmjs.com/package/uint8arrays) and given the fact the function is pretty simpler I think is not necessary to have an external lib for this.